gnome-extra/krb5-auth-dialog/krb5-auth-dialog-43.0.ebuild

```
@@ -14,7 +14,7 @@ SLOT="0"
 KEYWORDS="~amd64"
 
 RDEPEND="
-	app-crypt/gcr:=
+	app-crypt/gcr:=[gtk]
```

This alone does not fix the issue.  If you have only app-crypt/gcr:4 installed, the build still fails.

```suggestion
	app-crypt/gcr:0=[gtk]
```

---

Also needs a new revision (was automagic in use flag being off/on, imagine if on for host, not for binpkg consumer).

---

thanks!